config/locales/en.yml

```diff
@@ -986,6 +986,17 @@ en:
     no_records_unassigned_flash: "No items were unassigned from task"
     default_column: 'Name'
 
+  libraries:
+    sidebar:
+      title: "Libraries"
```

Comment:
Aren't they call "Inventories" now? :banana:

Reply:
at the time I had done this task they were "Libraries" 🍌